schemas/iso19139/src/main/plugin/iso19139/convert/thesaurus-transformation.xsl

```diff
@@ -229,7 +229,40 @@
       <!-- If no keyword, add one to avoid invalid metadata -->
       <xsl:if test="count(//keyword[thesaurus/key = $currentThesaurus]) = 0">
         <gmd:keyword gco:nilReason="missing">
-          <gco:CharacterString></gco:CharacterString>
+          <!-- Multilingual output if more than one requested language -->
```

Comment:
Do we need to build a complete empty keyword block for multilingual records ? The empty CharacterString was not good enough ?

Reply:
I added this mostly for the `gmx:Anchor` copying from a previous snippet, will check if for non anchor just adding `gco:CharacterString` works fine

Reply:
Simplified the template